strapi/strapi
# Fix issue 21333 The value displayed in the filter tag isn’t the same as the ones used to define it

### What does it do?

The value displayed in the filter tag isn’t the same as the ones used to define it 

### Why is it needed?

admin_user name was not displaying in a tag. When you tried to filter the list using admin_user it was not working

### How to test it?

- Create a Relation field with the admin user.
- Create a filter with the admin name
- Validate the filter
- See the result 

### Related issue(s)/PR(s)

#21333 

cc @simotae14
